src/lib/customPropTypes.js

```diff
@@ -251,6 +251,7 @@ export const contentShorthand = (...args) => every([
 export const itemShorthand = (...args) => every([
   disallow(['children']),
   PropTypes.oneOfType([
+    PropTypes.array,
```

Comment:
Fixes warnings with `tableFooter`.  This prop is shorthand for a Row but the array literal case is passed to the row cells via `mapValueToProps`.  Since the shorthand item factory supports arrays, the shorthand item type should as well.